docusaurus-plugin-moonwave/src/components/LuaType.js

```diff
@@ -181,18 +221,48 @@ function Token({ token, depth }) {
     case "union":
       return <Op>&nbsp;|&nbsp;</Op>
     case "luaType":
+      // Checks if the type is in the list of LuaClasses for the package
+      if (luaClassNames.includes(token.luaType.replaceAll("...", ""))) {
```

Comment:
We should extract the `token.luaType.replaceAll` value into a variable at the top of this case block, since we use it so much.

Instead of replacing `...` with `""`, why don't we just replace any non-alphanumeric or underscore with `""`? This way we can handle all possibilities.